sudi.go

```diff
@@ -156,7 +156,10 @@ func doSudiCert(VMname string) error {
 	if err != nil {
 		return err
 	}
-	pem.Encode(certPEM, &pem.Block {Type: "CERTIFICATE", Bytes: serverCert})
+	err = pem.Encode(certPEM, &pem.Block {Type: "CERTIFICATE", Bytes: serverCert})
```

Comment:
It would have been better to just do this in the first patch, but I guess it's not a big deal.

Reply:
true! I should have squashed them. Sorry.